packages/graphql-schema/src/directCashback/DirectCashbackRedemptionRequestListItemResolver.ts

```diff
@@ -0,0 +1,74 @@
+import { IDirectCashbackRedemptionRequestListInput } from "omnipartners";
+import { Arg, Args, ArgsType, Ctx, Field, Query, Resolver } from "type-graphql";
+import { ConnectionArgs } from "../connections";
+import { Context } from "../types/Context";
+import { GenericValidationError } from "../types/GenericValidationError";
+import {
+  DirectCashbackRedemptionRequestConnection,
+  DirectCashbackRedemptionRequestListItem,
+} from "./DirectCashbackRedemptionRequestListItem";
+
+@ArgsType()
+class DirectCashbackRedemptionRequestListInput {
+  @Field({ nullable: true })
+  public status?: string;
+
+  @Field({ nullable: true })
+  public barcode: string;
+
+  @Field({ nullable: true })
+  public sortField: string;
+
+  @Field({ nullable: true })
+  public sortOrder: string;
+
+  public toOmnipartners(): Omit<
+    IDirectCashbackRedemptionRequestListInput,
+    "user_guid"
+  > {
+    return {
+      status: this.status,
+      barcode: this.barcode,
+      sort_field: this.sortField,
+      sort_order: this.sortOrder,
+    };
+  }
+}
+
+@Resolver(of => DirectCashbackRedemptionRequestListItem)
```

Comment:
Tu peux enlever le `of => DirectCashbackRedemptionRequestListItem`. Le `of => Bar` fait référence au https://typegraphql.ml/docs/resolvers.html#field-resolvers et je ne pense pas que c'est le cas ici.